README.md

```diff
@@ -195,7 +195,7 @@ app = Chalice(app_name="hello-world")
 def index():
     return {"hello": "world"}
 
-app = epsagon.lambda_wrapper(app)
+app = epsagon.chalice_wrapper(app)
```

Comment:
It looks like this line must be before defining the route functions - otherwise, I don't see how it could make any effect since `app` is no longer used.

Reply:
It's the opposite - user defines everything on the app, then we just wrap it for the invocation calls.